server/dto/node_unregister_request.go

```diff
@@ -0,0 +1,6 @@
+package dto
+
+type NodeUnregisterRequest struct {
```

Comment:
ProposalUnregisterRequest ?

Reply:
`NodeUnregisterRequest` is ok also, it's more consistent with remote endpoint naming

Reply:
Fine by me , until this structure not leaked anywhere else out of mysterium api scope